configure.py

```
@@ -2314,6 +2314,13 @@ def make_bin_override():
 configure_inspector(output)
 configure_section_file(output)
 
+# Apply V8 patches if not already applied
+applied_marker = Path('deps/v8/patches/applied')
+if not applied_marker.exists():
+  info('Applying V8 patches...')
+  subprocess.run(['bash', 'deps/v8/apply-patches.sh'], check=True)
```

Ruff (0.14.1)

Starting a process with a partial executable path

(S607)